test/envoye2e/stats_plugin/stats_test.go

```diff
@@ -340,79 +340,85 @@ func TestStatsGrpc(t *testing.T) {
 }
 
 func TestStatsGrpcStream(t *testing.T) {
-	params := driver.NewTestParams(t, map[string]string{
-		"MetadataExchangeFilterCode": "inline_string: \"envoy.wasm.metadata_exchange\"",
-		"StatsFilterCode":            "inline_string: \"envoy.wasm.stats\"",
-		"WasmRuntime":                "envoy.wasm.runtime.null",
-		"DisableDirectResponse":      "true",
-		"UsingGrpcBackend":           "true",
-		"StatsConfig":                driver.LoadTestData("testdata/bootstrap/stats.yaml.tmpl"),
-		"StatsFilterClientConfig":    driver.LoadTestJSON("testdata/stats/client_config_grpc.yaml.tmpl"),
-		"StatsFilterServerConfig":    driver.LoadTestJSON("testdata/stats/server_config_grpc.yaml.tmpl"),
-	}, envoye2e.ProxyE2ETests)
-	params.Vars["ClientMetadata"] = params.LoadTestData("testdata/client_node_metadata.json.tmpl")
-	params.Vars["ServerMetadata"] = params.LoadTestData("testdata/server_node_metadata.json.tmpl")
-	enableStats(t, params.Vars)
-	params.Vars["ClientHTTPFilters"] = params.LoadTestData("testdata/filters/grpc_stats.yaml") + params.Vars["ClientHTTPFilters"]
-	params.Vars["ServerHTTPFilters"] = params.LoadTestData("testdata/filters/grpc_stats.yaml") + params.Vars["ServerHTTPFilters"]
+	env.SkipTSan(t)
+	for _, runtime := range Runtimes {
+		t.Run(runtime.WasmRuntime, func(t *testing.T) {
+			env.SkipV8(t, runtime.WasmRuntime)
```

Comment:
only test on `envoy.wasm.runtime.null` before, skip `envoy.wasm.runtime.v8`.

cc @kyessenov

Reply:
that's fine.